Source: Written by Niels Ole Salscheider
Upstream: No
Reason: This patch is necessary for the system libjpeg

diff -ur irrlicht-1.8.3.orig/source/Irrlicht/CImageLoaderJPG.cpp irrlicht-1.8.3/source/Irrlicht/CImageLoaderJPG.cpp
--- irrlicht-1.8.3.orig/source/Irrlicht/CImageLoaderJPG.cpp	2015-12-18 16:21:11.430458208 +0100
+++ irrlicht-1.8.3/source/Irrlicht/CImageLoaderJPG.cpp	2015-12-18 17:06:45.303483342 +0100
@@ -68,7 +68,7 @@
 boolean CImageLoaderJPG::fill_input_buffer (j_decompress_ptr cinfo)
 {
 	// DO NOTHING
-	return 1;
+	return TRUE;
 }
 
 
